rhel7/profiles/stig.profile

```diff
@@ -268,7 +266,6 @@ selections:
     - sshd_disable_compression
     - sshd_disable_user_known_hosts
     - chronyd_or_ntpd_set_maxpoll
-    - configure_firewalld_rate_limiting
```

Comment:
this has a historical event: https://github.com/ComplianceAsCode/content/pull/5168

Reply:
@vojtapolasek @ggbecker Please remove it as it's no longer relevant. If we want to re-add this, we need to go through the proper channels and communicate this to DISA.